### PR TITLE
Fix trace analytics navigaiton

### PR DESCRIPTION
**Pull request by ps48**
### Description
Fix trace Analytics and add side nav for child components

### Issues Resolved
#350 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/trace_analytics/trace_side_nav.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  EuiPage,
+  EuiPageBody,
+  EuiPageSideBar,
+  EuiSideNav,
+  EuiSideNavItemType,
+  EuiSwitch,
+} from '@elastic/eui';
+import React from 'react';
+
+export function TraceSideBar(props: { children: React.ReactNode }) {
```
Comment (joshuali925) [inline public/components/trace_analytics/trace_side_nav.tsx:19]: `ObservabilitySideBar` doesn't work?

**Review by joshuali925 — APPROVED** [review]

**Review by pjfitzgibbons — COMMENTED** [review]
When opening the Traces module, the Dashboard is displayed.
What would be displayed on Dashboard?
How is that different than the Jaeger or Services views ?

**Comment by ps48** [comment]
> When opening the Traces module, the Dashboard is displayed. What would be displayed on Dashboard? How is that different than the Jaeger or Services views ?

Trace Analytics by default shows a Dashboards with has an accumulated view of all traces and services. `Jaeger` and `Data Prepper` are sources for traces. The Traces view and Services show trace group and services drill down respectively. 

The overall functionality doesn't change here. It is the same as it was earlier. More info [here](https://opensearch.org/docs/1.2/observability-plugin/trace/ta-dashboards/)

File: public/components/trace_analytics/trace_side_nav.tsx (hunk 1 above)
Comment (ps48) [reply public/components/trace_analytics/trace_side_nav.tsx:19]: @joshuali925 It had some issues with home page so I just added this part to keep things simple. Will remove observability side bar later.

**Pull request merged**